addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -109,6 +109,7 @@ export class ReceiptScreen extends Component {
             {
                 data: this.pos.orderExportForPrinting(this.pos.get_order()),
                 formatCurrency: this.env.utils.formatCurrency,
+                screenName: "RecieptScreen",
```

Comment:
```suggestion
                screenName: "ReceiptScreen",
```

Reply:
Hi @caburj ,
I have changed `RecieptScreen` to `ReceiptScreen`
Can you please check it again.